### ISSUE TITLE
[a11y] Gestures are not intuitive for CollectionView and CarouselView

### ISSUE THREAD

**Issue by rachelkang** [rachelkang/recipeSearch#5]
When a screen reader is activated, clicking into an item should be possible with a double tap (as with other interactive elements)
However, clicking into an item currently requires a double tap _long press_ which is unconventional/unexpected

**Comment by rachelkang** [comment]
Explore [AccessibilityActions](https://developer.android.com/reference/android/view/accessibility/AccessibilityNodeInfo.AccessibilityAction) further
